sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -154,10 +155,29 @@ export abstract class SFTPIntegrationBase<DownloadType> {
           return false;
         }
       }
-      // Read all the file content and create a buffer with 'ascii' encoding.
-      const fileContent = await client.get(remoteFilePath, undefined, {
-        readStreamOptions: { encoding: FILE_DEFAULT_ENCODING },
-      });
+      let fileContent: string | NodeJS.WritableStream | Buffer;
+      const fileExtension = path.parse(remoteFilePath).ext.toLocaleLowerCase();
```

Comment:
Any particular reason to use `toLocaleLowerCase` instead of the regular `toLowerCase`?

Reply:
That was a mistake 😂 intention was `toLowerCase`

Reply:
As an alternative, the below was used in the past.
```ts
path.extname(file.originalname).toLowerCase()
```